tests/unit/test_token_management.py

```diff
@@ -0,0 +1,48 @@
+
+import spacy
+nlp = spacy.load("en_core_web_sm")
+
+from sem_covid.entrypoints.notebooks.topic_modeling.topic_modeling_wrangling.token_management import *
+
+
+def test_filter_stop_words():
+    text = 'lorem ipsum and some stuff'
+    doc = nlp(text)
+    doc_without_stop_words = filter_stop_words(doc)
+
+    assert list == type(doc_without_stop_words)
+    assert len(doc_without_stop_words) == 3
+    assert doc_without_stop_words[0].text == "lorem"
```

Comment:
you don't care about the order of elements, you simply need to assert here that there is no `and` in the list of tokens.

Reply:
I tried to compare the lists of tokens, but even there are the same it gives `False` statement. It's the the best way to test these functions, but at the time I didn't find something better